experimental/generation/generator/packages/library/templates/standard/en-us/form-DisplayOperation.en-us.lg.lg

```diff
@@ -8,15 +8,23 @@
 
 ${'#'} displayOperation(op, property, entity)
 - IF: \${op && property && entity.value} 
-        - \${operationName(op)} "\${entity.text}" to \${propertyName(property)}
+        - \${operationName(op)} \${formatValue(entity)} to \${propertyName(property)}
 - ELSEIF: \${op && property}
         - \${operationName(op)} \${propertyName(property)}
 - ELSE:
         - \${operationName(op)}
 
 ${join(flatten([namePlusSwitch(), cases(), default()]), EOL())}
+
+${'#'} formatValue(entity)
+- IF: \${isArray(entity.value)}
+        - \${entityValue(entity)}
+- ELSE:
+       - \${template(`\${entity.name}_Value`, entity.value)}
 ```
```

Comment:
I don't think you need this template--it should be handled through entityValue to the underlying type_Value template.